src/runtime/button-api-test.js

```diff
@@ -187,21 +188,42 @@ describes.realWin('ButtonApi', {}, env => {
 
   it('should log button click on create.', async () => {
     const button = buttonApi.create(handler);
-    eventManagerMock
-      .expects('logSwgEvent')
-      .withExactArgs(AnalyticsEvent.ACTION_SWG_BUTTON_CLICK, true)
-      .once();
-    button.click();
+    let count = 0;
+    sandbox
+      .stub(ClientEventManager.prototype, 'logSwgEvent')
+      .withArgs(AnalyticsEvent.ACTION_SWG_BUTTON_CLICK, true)
+      .callsFake(() => count++);
+    await button.click();
+    expect(count).to.equal(1);
   });
 
   it('should log button click on attach.', async () => {
     const button = doc.createElement('button');
     buttonApi.attach(button, {}, handler);
+    let count = 0;
+    sandbox
+      .stub(ClientEventManager.prototype, 'logSwgEvent')
+      .withArgs(AnalyticsEvent.ACTION_SWG_BUTTON_CLICK, true)
+      .callsFake(() => count++);
+    await button.click();
+    expect(count).to.equal(1);
+  });
+
+  it('should log button impression on create', async () => {
```

Comment:
And this is because create calls attach. yes?

Reply:
Yes that is correct. I just wanted to test both functions explicitly.